02_bubble/CameronSchafer/bubble_test.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+//main() test
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote("[1 2 3 5]\n[1 2 3 5]\n[1 2 3 5]\n")
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Errorf("Unexpected output in main()\nexpected: %q\nactual: %q", expected, actual)
+	}
+}
+
+//bubble() test
+func TestBubbleOutput(t *testing.T) {
+	//test cases with descriptions.
+	testCases := []struct {
```

Comment:
Consider using maps for your testCases with description as the key (see [Dave Cheney post](https://dave.cheney.net/2019/05/07/prefer-table-driven-tests))

Reply:
done. 
Updated to use maps